hal/armv7m/imxrt/10xx/hal.c

```diff
@@ -96,8 +97,23 @@ addr_t hal_kernelGetAddress(addr_t addr)
 }
 
 
+int hal_memAddMap(addr_t start, addr_t end, u32 attr, u32 id)
+{
+	return mpu_mapAlloc(start, end, attr, id);
+}
+
+
+void hal_memInfo(void)
+{
+	mpu_mapInfo();
+}
+
+
 int hal_cpuJump(addr_t addr)
 {
+	if (mpu_syspageConfig() < 0)
+		return -1;
+
```

Comment:
In command `go` implementation there is invoked function `devs_done()` before calling `hal_cpuJump`. 
If `hal_cpuJump` can return with false, the devices should be initialized again but I am not sure that it is a good approach.

https://github.com/phoenix-rtos/plo/blob/48f0255fde805aa552ffe4240ee1ec9b3127575b/hal/armv7m/mpu.c#L298-L306

Is it necessary to check these errors here?

Reply:
Maybe, all checkings there shall be removed. This dates the time of my draft from march when MPU regions were loaded in `plo`, and MPU enabled  right before jump to kernel text. It may now be unnecessary,to stop `go!`, although the kernel should then validate `syspage_hal_t`  content and take action if it finds an inconsist or missing MPU setup when expected.

Reply:
Removed.